flake.nix

```
@@ -0,0 +1,25 @@
+{
+  inputs.flake-utils.url = github:numtide/flake-utils;
+  outputs = { self, nixpkgs, flake-utils }:
+  flake-utils.lib.eachSystem [ "x86_64-linux" ] (system:
```

quick-lint-js should work on every platform:
```suggestion
  flake-utils.lib.eachSystem flake-utils.lib.allSystems (system:
```